File: SCKTester/TestUser.m
//
//  TestUser.m
//  ScheduleKit
//
//  Created by Guillem on 12/1/15.
//  Copyright (c) 2015 Guillem Servera. All rights reserved.
//

#import "TestUser.h"

@implementation TestUser

- (instancetype)initWithName:(NSString*)name color:(NSColor*)color {
    self = [super init];
    if (self) {
        _name = name;
        _labelColor = color;
    }
    return self;
}
@end

File: SCKExample-Swift/TestUser.swift
//
//  TestUser.swift
//  ScheduleKit
//
//  Created by Guillem Servera Negre on 2/11/2016.
//  Copyright © 2016-2017 Guillem Servera. All rights reserved.
//

import ScheduleKit
import Cocoa

@objcMembers final class TestUser: NSObject, SCKUser {

    var name: String

    var eventColor: NSColor

    init(name: String, color: NSColor) {
        self.name = name
        self.eventColor = color
        super.init()
    }
}
